packages/runtime/src/templates/getHandler.ts

```diff
@@ -22,17 +19,17 @@ const {
   getMaxAge,
   getMultiValueHeaders,
   getPrefetchResponse,
-  getNextServer,
   normalizePath,
 } = require('./handlerUtils')
+const { NetlifyNextServer } = require('./server')
 /* eslint-enable @typescript-eslint/no-var-requires */
 
 type Mutable<T> = {
   -readonly [K in keyof T]: T[K]
 }
 
 // We return a function and then call `toString()` on it to serialise it as the launcher function
-// eslint-disable-next-line max-params
+// eslint-disable-next-line max-params, max-lines-per-function
 const makeHandler = (conf: NextConfig, app, pageRoot, staticManifest: Array<[string, string]> = [], mode = 'ssr') => {
```

Comment:
Maybe for another PR, but this looks like this could be broken up as I see we disable the ESLint rule `max-lines-per-function`.

Reply:
yes 100%, @ascorbic and I discussed this too - it would be good to refactor this handler so that it is more like the edge handler and is a module itself, rather than a string output - then we can refactor it more easily into manageable chunks

Reply:
Cool cool. I'll create an issue for it.